nselib/http.lua

```diff
@@ -87,13 +87,20 @@
 --   end
 --   </code>
 -- * <code>no_cookie_overwrite</code>: Do not overwrite the cookies initially provided and when a duplicate cookie is received, ignore it.
--- * <code>enable_cookies</code>: Cookies are accessible to other http calls
--- The following example code demonstrates how the cookies can be passed and updated for every subsequent http calls.
---
+-- * <code>enable_cookies</code>: Cookies are accessible to other http calls.
+-- The following example code demonstrates how the cookies can be passed.(enable_cookies options automatically gets enabled if we pass the cookie jar)
 -- <code> 
--- http.get(host, port, {no_cookie_overwrite=false, cookies = options.cookies, enable_cookie = true})
--- </code
---
+-- http.get(host, port, {cookies = options.cookies})
+-- </code>
+-- The following example code demonstrates how the cookies can be passed and if we dont want the passed cookies to be overwritten on receiving the same cookie
+-- jar in the next request.
+-- <code> 
+-- http.get(host, port, {no_cookie_overwrite=false, cookies = options.cookies})
+-- </code>
+-- The following example code demonstrates how to get the cookies received in redirected request to come in the cookie jar
```

Comment:
Where are we receiving the cookies? The wording here is a bit off if we look at the command below.